Hazel/Dtls/DtlsConnectionListener.cs

```diff
@@ -17,7 +17,7 @@ namespace Hazel.Dtls
     /// <inheritdoc />
     public class DtlsConnectionListener : ThreadLimitedUdpConnectionListener
     {
-        const int MaxDatagramSize = 1200;
+        private const int MaxCertFragmentSize = 600;
```

Comment:
Wonder if we should just set to the expected minimum of 576 bytes while we are at it

Reply:
I was wrong and the Dtls headers are accounted for, so I dropped this to 550 to leave a little room for maybe something I'm not remembering.